bsonkit/convert.go

```diff
@@ -15,6 +15,23 @@ func Convert(m bson.M) Doc {
 	return &d
 }
 
+// Convert will convert a simple array to a list.
+func ConvertArrayOfDocumentsToList(a bson.A) List {
```

Comment:
Since we already use the `*List()` function naming throughout the package I think `ConvertList` would be more fitting.

Also, since this method is mainly used for testing, would `[]bson.M` maybe make more sense as the argument type?

Reply:
I think it's a better idea.

Reply:
Done